modules/elt/pages/use-airbyte-to-load-data-from-external-sources-to-teradata-vantage.adoc

```diff
@@ -0,0 +1,200 @@
+= Use Airbyte to load data from external sources to Teradata Vantage
+:experimental:
+:page-author: Krutik Pathak
+:page-email: krutik.pathak@teradata.com
+:page-revdate: June 6th, 2023
+:description: Use Airbyte with Teradata Vantage.
+:keywords: airbyte, data warehouses, compute storage separation, teradata, vantage, cloud data platform, object storage, data transfer, data extraction, enterprise analytics, elt.
+:tabs:
+:dir: getting-started-with-airbyte
+
+== Overview
+
+This tutorial showcases how to use Airbyte (an Open Source Extract Load Transform tool) with Teradata Vantage. We work with a very simple end-to-end setup to load the employees’ data from Google Sheets to Teradata Vantage using Airbyte.
```

Comment:
This was an edit regarding consistency in the capitalization that Adam pointed out.
```suggestion
This tutorial showcases how to use Airbyte (an open-source Extract Load Transform tool) with Teradata Vantage. We work with a very simple end-to-end setup to load the employees’ data from Google Sheets to Teradata Vantage using Airbyte.
```